docs/modules/ROOT/pages/distribution.adoc

```diff
@@ -1,38 +1,34 @@
 = Distribution
-:url-maven-artifact-query: http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22org.asciidoctor%22%20AND%20a%3A%22asciidoctorj%22%20AND%20v%3A%22{artifact-version}%22
-:url-maven-artifact-detail: http://search.maven.org/#artifactdetails%7Corg.asciidoctor%7Casciidoctorj%7C{artifact-version}%7Cjar
-:url-maven-artifact-file: http://search.maven.org/remotecontent?filepath=org/asciidoctor/asciidoctorj/{artifact-version}/asciidoctorj-{artifact-version}
-:url-maven-artifact-api-query: http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22org.asciidoctor%22%20AND%20a%3A%22asciidoctorj-api%22%20AND%20v%3A%22{artifact-version}%22
-:url-maven-artifact-api-detail: http://search.maven.org/#artifactdetails%7Corg.asciidoctor%7Casciidoctorj-api%7C{artifact-version}%7Cjar
-:url-maven-artifact-api-file: http://search.maven.org/remotecontent?filepath=org/asciidoctor/asciidoctorj-api/{artifact-version}/asciidoctorj-api-{artifact-version}
+
+:asciidoctorj-group-id: org.asciidoctor
+:url-maven-artifact-query: https://central.sonatype.com/artifact/org.asciidoctor/asciidoctorj/{artifact-version}/versions
+:url-maven-artifact-detail: https://central.sonatype.com/artifact/org.asciidoctor/asciidoctorj/{artifact-version}
+:url-maven-artifact-api-query: https://central.sonatype.com/artifact/org.asciidoctor/asciidoctorj-api/{artifact-version}/versions
+:url-maven-artifact-api-detail: https://central.sonatype.com/artifact/org.asciidoctor/asciidoctorj-api/{artifact-version}
 
 AsciidoctorJ is published to Maven Central.
 The artifact information can be found in the tables below.
 
-[cols="2,2,^2,4"]
+[cols="2,2,^2"]
 .Artifact information for AsciidoctorJ in Maven Central
 |===
-|Group Id |Artifact Id |Version |Download
```

Comment:
I am not sold on the idea of removing `Download` column. It's handy, specialty for anyone wanting to install the cli manually. Any other opinion?

Reply:
I interpreted your
> I personally like them, but I admit only the distribution is useful.
as "Go ahead and remove it" 😃 . 
Maybe I misunderstood, and you meant that the download link to the distribution (but not javadoc, for instance) is relevant?

Reply:
Tbh I'd also prefer having the link to the distribution in the docs.
There might be folks who want the distribution but don't want to install via sdkman or chocolatey.
Also folks coming from 
https://github.com/asciidoctor/asciidoctorj/blob/dd6008a0c9c95dac271470a36af2914d7d811074/docs/modules/ROOT/pages/cli.adoc#L13
might be looking for some hint on a tar or a zip.

Reply:
> as "Go ahead and remove it"

It was exposing a "pro" and a "con" for keeping it, I don't have a strong opinion. If anything I'd like to hear @robertpanzer opinions since he's the main maintainer.

To me the decision comes down to the usefulness of the information for anyone hitting the repo (specially first-timers). All the info can be found in the maven central links already. And in the case of making CLI installation more straightforward (to save some extra clicks), we could remove the "Download" column and add a section below for sdkman. I see we have one for Windows, but nothing for *nix systems.

Reply:
Added the Download column again, adjusted API calls to use `https` instead of `http`, so now they work again. 
I did, however, remove the link to the `tar`, as there is no `tar`.